source/Octopus.Tentacle.Contracts/KubernetesScriptServiceV1Alpha/StartKubernetesScriptCommandV1Alpha.cs

```diff
@@ -51,5 +53,7 @@ public class StartKubernetesScriptCommandV1Alpha
         public Dictionary<ScriptType, string> Scripts { get; } = new();
         public List<ScriptFile> Files { get; } = new();
         public string[] Arguments { get; }
+
+        public string? ScriptPodServiceAccountName { get; }
```

Comment:
Because this an addition of a nullable/option field, this is a backwards compatible change with existing agents